tests/ui/cbmc_checks/float-overflow/expected

```diff
@@ -0,0 +1,10 @@
+Failed Checks: NaN on +
```

Comment:
Did you find a way to check that `var_n` does not appear in the output? If not, we can add support for a `^<pattern>` to `compiletest` which checks that `<pattern>` does not appear in the output.

Reply:
> BTW, I was actually thinking we might as well replace "+" by addition and so on to keep the messages consistent and more readable. Any thought?

I like that!